### PR TITLE
#316 add 19-20 DB migration

### LINKED ISSUE #316: Write a new database migration

### PR DESCRIPTION
**Pull request by zuevval**
closes #316 

This request would be logically correct to merge after resolving #314, #323 and #324

### CODE REVIEW

File: app/src/main/java/com/github/braillesystems/learnbraille/data/db/LearnBrailleDatabase.kt [hunk 1]
```diff
@@ -209,119 +210,132 @@ private val MIGRATION_17_18 = object : Migration(17, 18) {
...
         }
+    }
+
+    Timber.i("New data inserted")
+}
 
+private val MIGRATION_18_19 = object : Migration(18, 19) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        Timber.i("Start 18-19 migration")
+        updateTheoryAndMaterials(database = database)
```
Comment (winter-yuki) [inline app/src/main/java/com/github/braillesystems/learnbraille/data/db/LearnBrailleDatabase.kt:329]: It is not needed here to provide named arguments

**Review by winter-yuki — APPROVED** [review]
Looks ok. But I have no time to test manually myself.

File: app/src/main/java/com/github/braillesystems/learnbraille/data/db/LearnBrailleDatabase.kt (hunk 1 above)
Comment (zuevval) [reply app/src/main/java/com/github/braillesystems/learnbraille/data/db/LearnBrailleDatabase.kt]: fixed

**Comment by zuevval** [comment]
> Looks ok. But I have no time to test manually myself.

I've tested. It works. Trust me 🤞🏼

**Pull request merged**